pkg/expression/exprsession/sessionctx.go

```diff
@@ -40,126 +40,126 @@ import (
 	"go.uber.org/zap"
 )
 
-// SessionExprContext should implement the `ExprContext` interface.
-var _ exprctx.ExprContext = &SessionExprContext{}
+// ExprContext should implement the `ExprContext` interface.
+var _ exprctx.ExprContext = &ExprContext{}
 
-// SessionExprContext implements `ExprContext`
-type SessionExprContext struct {
+// ExprContext implements `ExprContext`
+type ExprContext struct {
 	sctx sessionctx.Context
 	*SessionEvalContext
 }
 
-// NewSessionExprContext creates a new SessionExprContext.
-func NewSessionExprContext(sctx sessionctx.Context) *SessionExprContext {
```

Comment:
maybe name the pkg as `sessionexpr`, expression and exprsession is so similar

then we can rename `NewSessionExprContext` as `Context`